pages/verksamhetsomraden.html

```diff
@@ -1,91 +1,180 @@
         </p>
-    </div>
-    <div class="container">
-        <img src="/assets/images/bilder/YH-start.png" alt="yrkeshögskolan" >
-        <br>
-        <a href="#">Yrkeshögskolan</a>
-        <p>Utbildningar som svarar mot arbetslivets behov av kvalificerad yrkeskompetens och som drivs i nära samverkan med företag och andra arbetsgivare.</p>
-    </div>
-    <div class="container">
-        <img src="/assets/images/bilder/KK-start.png" alt="yrkeshögskolan">
-        <a href="#">Konst och kulturutbildningar</a>
-        <p>Utbildningar som är högskoleförberedande, yrkesinriktade eller inom kulturarvsområdet och drivs av privata utbildningsanordnare.</p>
-    </div>
-    <div class="container">
-        <img src="/assets/images/bilder/Kontakttolk.png" alt="yrkeshögskolan">
-        <a href="#">Tolkutbildningar</a>
-        <p>Utbildningar till kontakttolk, teckenspråks- och dövblindtolk samt skrivtolk som drivs av folkhögskolor och studieförbund.</p>
-    </div>
-    <div class="container">
-        <img src="/assets/images/bilder/ET-start.png" alt="yrkeshögskolan">
-        <a href="#">Utbildningar med endast tillsyn</a>
-        <p>Eftergymnasiala utbildningar inom olika områden som bedrivs av fristående skolor. Utbildningarnas statliga stöd omfattas endast av tillsyn.</p>
-    </div>
-    <div class="container">
-        <img src="/assets/images/bilder/SEQF.png" alt="yrkeshögskolan">
-        <a href="#">SeQF</a>
-        <p>Sveriges referensram för kvalifikationer (SeQF) ska synliggöra allt lärande för att öka matchningen på arbetsmarknaden.</p>
-    </div>
-    <div class="container">
-        <img src="/assets/images/bilder/Scandinav_41YW_validering_webb.jpg" alt="yrkeshögskolan">
-        <a href="#">Om validering</a>
-        <p>Validering innebär att en persons kunskaper och kompetenser kartläggs och bedöms på ett strukturerat sätt, för studier eller arbete, oavsett hur, var eller när de förvärvats.</p>
-    </div>
+      </article>
+
+      <section id="main-gateway" class="container pl-0 pr-0">
+        <section>
+          <article class="background-dark-slate-gray-l-65">
+            <img
+              src="/assets/images/bilder/YH-start.png"
+              style="width: 100%"
+              alt="yrkeshögskolan"
+            />
+            <a href="#">Yrkeshögskolan</a>
+            <p>
+              <br />
+              Utbildningar som svarar mot arbetslivets behov av kvalificerad
+              yrkeskompetens och som drivs i nära samverkan med företag och
+              andra arbetsgivare.
+            </p>
+          </article>
+        </section>
+
+        <section>
+          <article class="background-dark-slate-gray-l-65">
+            <img
+              src="/assets/images/bilder/KK-start.png"
+              style="width: 100%"
+              alt="yrkeshögskolan"
+            />
+            <a href="#">Konst och kulturutbildningar</a>
+            <p>
+              <br />
```

Comment:
@Oskar-Mikael you are correct in that br tags should be avoided. In any regular body of text, normally, you would have paragraphs for each block of text, not a bunch of line breaks 😉 

I did set up a margin and padding system in our styles at the beginning, using **m-, mt-, mr-, mb- and ml-** (_margin, margin-top, margin-right etc_) and the **same pattern for padding** (_using p instead of m_) followed by a **number between 0-5**. This will allow us to have more standardised spacing in our layout.

- **(m|p)[position]-[number]:** using m or p only, will cover top, right, bottom and left, **i.e. (margin|padding): x x x x;**
- Measurement unit used is rem, **e.g**. `class="pb-3"` **=** `padding-bottom: 3rem !important;` 

**Usage example:**
```
<p class="mb-0">Ridiculus Commodo</p>
<p>Ridiculus Commodo</p>
```
This would remove the bottom margin of the first paragraph, effectively creating the same visual effect as the `<br />` tag 🤓